tests/docker/ducker-ak

```diff
@@ -58,10 +58,14 @@ help|-h|--help
     Display this help message
 
 up [-n|--num-nodes NUM_NODES] [-f|--force] [docker-image]
+        [-E|--custom-ducktape DIR]
```

Comment:
The E was chosen somewhat arbitrarily.  If anyone has a better suggestion that didn't conflict with an existing flag, I'd be happy to implement it